source/brailleDisplayDrivers/albatross/constants.py

```diff
@@ -0,0 +1,110 @@
+# A part of NonVisual Desktop Access (NVDA)
+# This file is covered by the GNU General Public License.
+# See the file COPYING for more details.
+# Copyright (C) 2021 NV Access Limited, Burman's Computer and Education Ltd.
+
+from typing import List
+
+BAUD_RATE = 19200
+TIMEOUT = 0.2
+WRITE_TIMEOUT = 0
+# How many times initial connection is waited by sleeping TIMEOUT.
+# It sounds big value but users should appreciate that braille starts.
+# It should not harm other devices with same PID&VID because this device
+# is last one.
+MAX_INIT_SLEEPS = 20
+# Maybe due to writings of other drivers with same PID&VID, device seems
+# to send init packets it should not. Although reset do not work well,
+# hopefully at least strange packets would be discarded.
+# How many times to try to reset I/O buffers
+RESET_COUNT = 10
+# How long to sleep between I/O buffer resets
+RESET_SLEEP = 0.05
+WRITE_QUEUE_LENGTH = 20
+KEY_NAMES = {

```

Comment:
This should probably be a `IntEnum`

class Keys(IntEnum):
	attribute1: 1
	...
```

Reply:
> This should probably be a `IntEnum`
> 
> ```python
> class Key(IntEnum):
> 	attribute1: 1
> 	...
> ```

I think KEY_NAMES is the same solution other drivers use, so I think to keep it.

Reply:
Please make this change or this PR will not be accepted

Reply:
> Please make this change or this PR will not be accepted

You surely have good reason to say that but unfortunately I don't now or understand it.

Reply:
There's a few material benefits to the data structure.

At core is that it is a more accurate abstraction pattern.
For example, there is nothing that connects the shared `1` between `KEY_NAMES` `CONTROL_KEY_CODES`.
Assigning 1 to a value creates a common abstraction between keys, key names and control key codes.

Many of the braille display drivers in NVDA are legacy code. The seika notetaker driver and the braille HID driver are the only ones I am aware of added since the Python 3 upgrade.
If you look at other newly added NVDA code you will notice abstraction patterns like this and improved code styling standards.